### PR TITLE
Readable directory should not be handled as a file

### PR DESCRIPTION
**Pull request by theonefx**
in some container, a folder represented by resource is readable;
this will course some exception；

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@theonefx Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=spring-io/initializr&pullRequestId=1061)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=spring-io/initializr&pullRequestId=1061) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@theonefx Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=spring-io/initializr&pullRequestId=1061)!

File: initializr-generator/src/main/java/io/spring/initializr/generator/project/contributor/MultipleResourcesProjectContributor.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void contribute(Path projectRoot) throws IOException {
 		Resource root = this.resolver.getResource(this.rootResource);
 		Resource[] resources = this.resolver.getResources(this.rootResource + "/**");
 		for (Resource resource : resources) {
-			if (resource.isReadable()) {
```
Comment (snicoll) [inline initializr-generator/src/main/java/io/spring/initializr/generator/project/contributor/MultipleResourcesProjectContributor.java]: Resources aren't file when they're packaged in a JAR file and that check is breaking the build. Please run the build prior to submitting a PR.

**Comment by snicoll** [comment]
@theonefx that change isn't fixing the problem you're facing. Can you please update your PR or describe the problem you're facing in more details?

**Pull request closed without merging**